IntegrationTests/Tests/IntegrationTests/XCBuildTests.swift

```diff
@@ -1,40 +1,41 @@
 /*
  This source file is part of the Swift.org open source project
 
- Copyright (c) 2014 - 2020 Apple Inc. and the Swift project authors
+ Copyright (c) 2014 - 2025 Apple Inc. and the Swift project authors
  Licensed under Apache License v2.0 with Runtime Library Exception
 
  See http://swift.org/LICENSE.txt for license information
  See http://swift.org/CONTRIBUTORS.txt for Swift project authors
  */
 
-import XCTest
 import TSCBasic
 import TSCTestSupport
+import XCTest
```

Comment:
issue (blocking): Since we converted to swift testing, can we remove this import and replace all XCTest API with the associated Swift Testing call?

See https://developer.apple.com/documentation/testing/migratingfromxctest#Record-issues